src/main/java/net/apnic/whowas/loaders/config/LoaderConfiguration.java

```diff
@@ -105,10 +110,19 @@ private void buildTree()
     }
 
     @PostConstruct
```

Comment:
Note that spring says @Transactional should not be relied upon in @PostConstruct as the proxy may not be fully initialised.

https://docs.spring.io/spring/docs/5.0.3.BUILD-SNAPSHOT/spring-framework-reference/data-access.html#transaction-declarative-annotations